pages/Technical Documentation/Data/Contributing to the EarthCODE Catalog.md

```diff
@@ -242,16 +252,383 @@ Alternatively, if you have access to a **reliable cloud storage service**—such
 
 If you choose to use the ESA PRR the generated STAC Items have to be sent via `POST` requests to the [PRR Registration Gateway](https://eoresults.esa.int/reg-api/docs#/) for registration using the previously requested collection.
 
-The only requirement is that __other users and the STAC browser must be able to find and read your repository!__
+The only requirement is that **other users and the STAC browser must be able to find and read your repository!**
 
 :::info  
 In the next step where you will be uploading metadata to the Open Science Catalog, EarthCODE administrators will review your Item Catalog and assist you with any necessary adjustments.  
-:::  
+:::
+
+## Step 3: Creating a Product Entry in the Open Science Catalog
+
+After completing steps 1 and 2 to prepare your data package, you're now ready to add a product entry to the Open Science Catalog (OSC).
```

Comment:
I propose to extend this paragraph with: "This can be done by using the [Catalog Editor](TBD) in the EarthCODE portal or by using Git directly as described below."